cli/cmd/service/update.go

```diff
@@ -0,0 +1,62 @@
+package service
+
+import (
+	"encoding/json"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func UpdateCmd(ch *cmdutil.Helper) *cobra.Command {
+	var newName string
+	var attributes string
+
+	updateCmd := &cobra.Command{
+		Use:   "update <service-name>",
```

Comment:
Call the command `edit` instead of `update` to align with other commands for updates/edits